openff/toolkit/_tests/test_molecule.py

```diff
@@ -924,7 +924,39 @@ def test_unique_atom_names(self, molecule):
         assert (
             len(set([atom.name for atom in molecule.atoms])) == molecule.n_atoms
         ) == molecule.has_unique_atom_names
-        assert all("x" in a.name for a in molecule.atoms)
+        assert all(
+            a.name.endswith("x") for a in molecule.atoms
+        )  # fails for xenon, and nothing here checks the x is in the correct place
+        # generate_unique_atom_names tests
```

Comment:
Cleaning up some comments

```suggestion
        )  
```